default.yaml

```diff
@@ -188,3 +188,7 @@ kratosConfig:
 
 captcha:
   mandatory: false
+
+skipFeeProbe:
```

Comment:
why not just an array?

Reply:
I thought it would be useful to have some context if we had to add any other services.

Reply:
I would agree with @dolcalmi that just a series of keys would be enough.

you could have the name of the service in comments, or it's easy enough to fetch it otherwise.

ie: 
skipFeeProbe:
  - "038f8f113c580048d847d6949371726653e02b928196bad310e3eda39ff61723f6" # muun